mmv1/products/spanner/InstancePartition.yaml

```diff
@@ -0,0 +1,120 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'InstancePartition'
+description: |
+  A Cloud Spanner instance partition is a unit of Cloud Spanner database capacity
+  that can be used to partition data and processing capacity within an instance.
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/spanner/docs'
```

Comment:
Is there a specific doc for instance partition?

Reply:
Added